### DISCUSSION TITLE
Log Transform for Data

### DISCUSSION THREAD

**Discussion by mirfani28** [teamalgoritma/community#470] [Q&A] [answered]
Halo Teman-Teman dan Instruktur Algo

Saya beberapa waktu lalu menonton video ini : [log transform by StatQuest](https://www.youtube.com/watch?v=VSi0Z04fWj0&t=694s)
Dan minggu ini saya di materi Time Series mendapat salah satu contoh aplikasinya yaitu memproses data multiplikatif menjadi aditif (personally I think it's very clever).

Kira-kira apa lagi pemanfaatan log transform di data science?
Saya pernah baca untuk memproses data yang tidak berdistribusi normal (skewed left) juga bisa dilakukan log transform agar puncak menjadi lebih ke tengah. Apakah ini betul?

Terimakasih

**Comment by nisa-basalamah** [comment]
Dear @mirfani28 ,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima. Kami akan mencoba membantu mencarikan referensi berkaitan dengan pertanyaan yang diberikan.

**Comment by muhrisman** [comment, marked as answer]
Halo mas, izin bantu jawab

Log Transformation betul bisa juga digunakan untuk mengubah suatu distribusi data yang "tidak normal" menjadi "normal" atau seperti yang mas bilang, _puncaknyanya lebih ke tengah_

Namun perlu diingat jika Log transformation digunakan untuk kasus tertentu misal kasus analisis regresi, interpretasi yang dilakukan terhadap model persamaan yang didapatkan tidak bisa diterjemahkan langsung. Persamaan tersebut harus ditransformasi balik jika hendak diinterpretasikan. 

Berikut rujukan yang mungkin membantu:  https://link.medium.com/zqplQqNovvb

Semoga membantu :) 

Regards, 
Muhamad Risman. 
Algoritma student batch Kappa

**Reply by mirfani28** [reply]
Thanks a lot Mas :)

ditransform balik menggunakan exp() ya berarti hasil persamaannya kalau di R.

**Reply by muhrisman** [reply]
Benar mas pakai exp() karena fungsinya logaritmik
